source/includes/_egress.md

```diff
@@ -163,10 +164,11 @@ pausing the user payment and prompting manual intervention, potentially delaying
 
 ### Response JSON Fields
 
-Field       | Type    | Description
------------ | ------- | -----------
-invoice_id  | String  | Internal **ON/RAMP**'s Invoice Identifier.
-invoice_url | Url     | Url where to redirect user.
+Field          | Type    | Description
+-------------- | ------- | -----------
+invoice_id     | String  | Internal **ON/RAMP**'s Invoice Identifier.
+invoice_url    | Url     | Url where to redirect user.
+invoice_url_qr | Url     | Url where to redirect user in QR code format.
```

Comment:
```suggestion
invoice_url_qr | Url     | Source Url for an image with QR-encoded `invoice_url`
```

Reply:
The qr is not just the invoice but the whole user links that in turns contains the invoice.